change/@fluentui-react-charting-db3502d6-ad5b-4426-bcf7-9644b4a9db7b.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "multistack bar chart bug fix and vr-test integration",
```

Comment:
We need to add stories for following charts:
- Tree
- Grouped Vertical Bar
- Stacked Bar
- Legends